resources/scss/ceres/widgets/Category/_item-grid-widget.scss

```diff
@@ -42,8 +17,4 @@
             }
         }
     }
-
```

Comment:
![Bildschirmfoto 2020-02-10 um 16 23 44](https://user-images.githubusercontent.com/11943448/74162919-d8041780-4c21-11ea-9c6e-b58e33136e10.png)
Das ist mit Standard Paddings&Margins
Man sollte dann auch in der category-list-item.scss diese Zeile entfernen
`margin-bottom: $grid-gutter-width / 2;`